examples/frameworks/adk_demo/src/nat_adk_demo/location.py

```
@@ -0,0 +1,136 @@
+            continue
+
+        try:
+            payload = await asyncio.to_thread(fetch_json,
+                                              geocoding_url, {
+                                                  "name": candidate_query,
+                                                  "count": 1,
+                                                  "language": "en",
+                                                  "format": "json",
+                                              },
+                                              timeout_seconds)
+        except Exception as ex:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)